src/Tabs/Tab.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import { colors, theme } from '../theme.js'
+
+const Tab = ({ icon, onClick, selected, disabled, children }) => (
+    <button
+        className={`${cx({
+            selected,
+            disabled,
+        })}`}
+        onClick={disabled ? undefined : onClick}
+    >
+        {icon}
+        <span>{children}</span>
+
+        <style jsx>{`
+            button {
+                flex-grow: 0;
+                position: relative;
+                display: inline-flex;
+                justify-content: center;
+                align-items: center;
+                vertical-align: bottom;
+
+                height: 100%;
+                padding: 19px 16px 14px 16px;
+
+                background-color: transparent;
+                outline: none;
+                border: none;
+                border-bottom: 1px solid ${colors.grey400};
+
+                color: ${colors.grey600};
+                font-size: 14px;
+                line-height: 14px;
+
+                transition: all 150ms ease-in-out;
+                transition-property: color, background-color;
+                cursor: pointer;
+            }
+
+            :global(.fixed) > button {
```

Comment:
This definitely should be `.tabs.fixed`, otherwise `.fixed` class on a body/html element (to prevent scrolling) might cause this to behave differently

Reply:
I don't think your worries are valid, because this rule will end up like this:
`.fixed > button.jsx-123456` 
And please note the `>` [Child combinator](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator) in the selector.
So:
- This style is only applied to buttons with this unique class name, so no risk of styles leaking out
- This style is only applied when these buttons are **immediate children** of an element with the `.fixed` class.

Actually I initially went with the `.tab-bar.fixed > button` selector, but then remembered your slack suggestion about potentially using the `Tab` element outside of the `TabBar`, so this slightly more generic, but still secure selector seemed more appropriate.

Reply:
Oops you're right, how could I miss that... my bad

Reply:
NP 👍